lib/Byng/Pimcore/Sitemap/Generator/SitemapGenerator.php

```diff
@@ -128,16 +168,18 @@ private function getDateFormat($date)
     /**
      * Notify search engines about the sitemap update.
      *
+     * @param string $domain
      * @return void
      */
-    private function notifySearchEngines()
+    private function notifySearchEngines($domain = null)
     {
         $googleNotifier = new GoogleNotifier();
 
-        if ($googleNotifier->notify()) {
+        if ($googleNotifier->notify($domain = null)) {
```

Comment:
1. assign `null` to `$domain`
2. pass `$domain` (now `null`) in as a parameter to `notify`

Did you mean to add the ` = null` here?